internal/generator/config/curve.go

```diff
@@ -1,11 +1,12 @@
 package config
 
 import (
-	"math/big"
-
 	"github.com/consensys/gnark-crypto/field"
+	"math/big"
 )
 
+//TODO: Same data structure used in runtime and for code generation. HashInfo not needed for runtime
```

Comment:
yes, we should add (separate PR / ticket) build tags for the code generation ,to ensure we don't pollute API or binary with useless structs needed only for code gen.